src/camtasia/resources/new.cmproj/project.tscproj

```diff
@@ -3,26 +3,23 @@
   "description" : "",
   "author" : "",
   "targetLoudness" : -18.0,
-  "shouldApplyLoudnessNormalization" : false,
-  "videoPlaybackRate" : 30,
-  "audioPlaybackRate" : 44100,
-  "width" : 1280.0,
-  "height" : 720.0,
-  "version" : "3.0",
+  "shouldApplyLoudnessNormalization" : true,
+  "videoFormatFrameRate" : 30,
+  "audioFormatSampleRate" : 44100,
```

Comment:
This is something we'll need to think about: the "new project" in the master branch is Camtasia 2019, but you've updated it to Camtasia 2020. We probably want to support both, but I haven't given any real thought how to do that. Depending on how different the formats are, it could mean completely parallel sets of code for different versions.